tests/app/celery/test_scheduled_tasks.py

```diff
@@ -486,6 +487,28 @@ def test_get_dynamodb_comp_pen_messages_filters(dynamodb_mock, sample_dynamodb_i
     assert len(messages) == 2
 
 
+def test_it_update_dynamo_item_is_processed_updates_properly(dynamodb_mock, sample_dynamodb_insert):
+    items_to_insert = [
+        {'id': '1', 'is_processed': False, 'payment_id': 1, 'paymentAmount': Decimal(1.00)},
+        {'id': '2', 'is_processed': False, 'payment_id': 2, 'paymentAmount': Decimal(2.50)},
+        {'id': '3', 'is_processed': True, 'payment_id': 1, 'paymentAmount': Decimal(0.00)},
+    ]
+
+    # Insert mock data into the DynamoDB table.
+    sample_dynamodb_insert(items_to_insert)
+
+    with dynamodb_mock.batch_writer() as batch:
+        for item in items_to_insert:
+            _update_dynamo_item_is_processed(batch, item)
+
+    response = dynamodb_mock.scan()
+
+    # Ensure we get all 3 records back and they are set to is_processed == True
+    assert response['Count'] == 3
+    for item in response['Items']:
+        assert item['is_processed'] is True
```

Comment:
```suggestion
        assert item['is_processed']
```
Please search Pep8 for "Don’t compare boolean values to True or False using".

Reply:
changed